README.md

```diff
@@ -52,27 +59,26 @@ The main acceptance criteria of any submitted project would be its impact on the
 
 We would like to advise 3 following categories:
 
-**Industry Canary dApp**
+**Canary Dapp**
 
 A Canary dApp is one that could be deployed on to a value bearing network, but may not necessarily be battle hardened enough for serious use cases. They are meant to be a step beyond simple tutorials while still being self-contained enough to provide a useful reference for developers building production grade dApps.
 Some examples:
-- **Lending application** that allows users to earn interest on deposits and borrow assets with a variable or stable interest rate. (2 types)
+- **Lending application** that allows users to earn interest on deposits and borrow assets with a variable or stable interest rate.
 - **Liquid staking** a liquid staking provider takes token deposits, stakes those tokens, and gives the depositor a receipt which is redeemable for the staked tokens.
 - **Aggregator** that unites separate decentralized protocols and aggregates liquidity from a variety of decentralized exchanges to facilitate cost-efficient transactions.
 
-**Ink! Cutting-edge Showcase**
+**Technical Showcase**
 
-This category implies developing full-stack application that clearly demonstrates the advantages of ink! over other smart contract languages(e.g. Solidity). In this category you should focus on opening the full potential of ink. The case doesn't necessarily needs to be business oriented. However it should clearly and visually demostrate superiority of ink. 
+This category implies developing full-stack application that clearly demonstrates the advantages of ink! over other smart contract languages (e.g. Solidity). In this category you should focus on opening the full potential of ink. The case doesn't necessarily needs to be business oriented. However it should clearly and visually demostrate superiority of the ink! language.
 You can use following aspects as indicators:
-- gas
-- smart contract size (find a use case where the size of smart contract will be smaller then solidity - application with complex calculations)
-- code writing experience (how comfortable to write code)
-- debugging experience
-
+- efficient gas usage
+- decreased smart contract size
+- improved developer experience
```

Comment:
How can a "full-stack application" improve developer and debugging experiences? I think these should move to "Infrastructure"